src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -29,7 +37,7 @@
   }
 
   &.#{$prefix}-controlbar-visible {
-    bottom: 5em;
-    transition: bottom $animation-duration-short ease-in;
+    height: 70%;
```

Comment:
Why does the relative height here lead to an absolute positioning regarding the control bar, no matter the player height?